Google.Api.Generator.Utils/Roslyn/RoslynBuilder.cs

```diff
@@ -203,6 +203,19 @@ public static PropertyDeclarationSyntax AutoProperty(Modifier modifiers, TypeSyn
             ParameterListSyntax Params() => ParameterList(SeparatedList(Enumerable.Repeat(parameter, parameter == null ? 0 : 1)));
         };
 
+        public static CodeFunc<LambdaExpressionSyntax> Lambda(ParameterSyntax parameter1, ParameterSyntax parameter2) => code =>
```

Comment:
Can this be changed to accommodate any amount of parameters?

Reply:
Potentially, although I suspect it'll make the code more complicated - and I very much doubt that we'll need more than 2 parameters. (We've managed with just one for a long time.) I'd prefer to wait until there's a compelling reason to generalize before doing so.

Reply:
I can see someonw writing a 3-param version in the future, but I get your point.